### PR TITLE
CovidVaccineEquity

### PR DESCRIPTION
**Pull request by carterm**
Creates the data for `data/vaccine-equity/` in the static repo.

### CODE REVIEW

**Review by chachasikes — APPROVED** [review]
Awesome.

I like the formatting on the .sql file a lot.
What's nice about the individual lines are that it will help tracking any field level changes, which is a pattern that can be useful for tracking down changes over time.

File: CovidVaccineEquity/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const notifyChannel = 'C01AA1ZB05B';
+const debugChannel = 'C01DBP67MSQ'; // 'C01AA1ZB05B';
+
+
+module.exports = async function (context, myTimer) {
+  const appName = context.executionContext.functionName;
+  try {
+    await slackBotChatPost(debugChannel,`${appName} (Every 30 mins from 10:15am to 1:45pm)`);
+
+    //turning the whole thing off until it's fixed
```
Comment (aaronhans) [inline CovidVaccineEquity/index.js:12]: is this comment outdated?

**Review by aaronhans — APPROVED** [review]
Fantastic!

File: CovidVaccineEquity/index.js (hunk 1 above)
Comment (carterm) [reply CovidVaccineEquity/index.js]: Got it.  Also got the notify channel.

**Pull request merged**